bittensor/config.py

```diff
@@ -182,37 +139,21 @@ def __init__(
             default_param_args = [_config.get("command"), _config.get("subcommand")]
 
         ## Get all args by name
-        default_params = parser.parse_args(args=default_param_args)
+        parser_no_required = copy.deepcopy(parser)
+        for i in range(len(parser_no_required._actions)):
+            parser_no_required._actions[i].required = False
+        default_params = parser_no_required.parse_args(args=default_param_args)
 
         all_default_args = default_params.__dict__.keys() | []
         ## Make a dict with keys as args and values as argparse.SUPPRESS
         defaults_as_suppress = {key: argparse.SUPPRESS for key in all_default_args}
-        ## Set the defaults to argparse.SUPPRESS, should remove them from the namespace
-        parser_no_defaults.set_defaults(**defaults_as_suppress)
-        parser_no_defaults._defaults.clear()  # Needed for quirk of argparse
-
-        ### Check for subparsers and do the same
-        if parser_no_defaults._subparsers != None:
-            for action in parser_no_defaults._subparsers._actions:
-                # Should only be the "command" subparser action
-                if isinstance(action, argparse._SubParsersAction):
-                    # Set the defaults to argparse.SUPPRESS, should remove them from the namespace
-                    # Each choice is the keyword for a command, we need to set the defaults for each of these
-                    ## Note: we also need to clear the _defaults dict for each, this is a quirk of argparse
-                    cmd_parser: argparse.ArgumentParser
-                    for cmd_parser in action.choices.values():
-                        # If this choice is also a subparser, set defaults recursively
-                        if cmd_parser._subparsers:
-                            for action in cmd_parser._subparsers._actions:
-                                # Should only be the "command" subparser action
-                                if isinstance(action, argparse._SubParsersAction):
-                                    cmd_parser: argparse.ArgumentParser
-                                    for cmd_parser in action.choices.values():
-                                        cmd_parser.set_defaults(**defaults_as_suppress)
-                                        cmd_parser._defaults.clear()  # Needed for quirk of argparse
-                        else:
-                            cmd_parser.set_defaults(**defaults_as_suppress)
-                            cmd_parser._defaults.clear()  # Needed for quirk of argparse
+
+        def l_set_defaults(l_parser):
+            ## Set the defaults to argparse.SUPPRESS, should remove them from the namespace
```

Comment:
Convert this to a dostring. Also, please make the function name `l_set_defaults` a little more descriptive. Example: `load_set_defaults`.
Also, add type annotation.